jest.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     "^.+\\.vue$": "vue-jest",
     ".+\\.(css|styl|less|sass|scss|svg|png|jpg|ttf|woff|woff2)$":
       "jest-transform-stub",
-    "^.+\\.jsx?$": "babel-jest"
+    "^.+\\.(js|jsx)?$": "babel-jest"
```

Comment:
you don't want this included, right?

Reply:
better not. but will be overwritten by the mobile fixes anyway.